.github/dependabot.yml

```diff
@@ -136,6 +136,10 @@ updates:
     directory: "/docker/httpd"
     schedule:
       interval: "weekly"
+  - package-ecosystem: "docker"
+    directory: "/docker/httpd-discovery"
+    schedule:
+      interval: "weekly"
```

Comment:
Out of ignorance, how is this related to the rest of the PR?

Reply:
> Out of ignorance, how is this related to the rest of the PR?

Not sure, but the `gendependabot` job failed without it: https://github.com/signalfx/splunk-otel-collector/actions/runs/10426155220/job/28878539012